ansible/roles/image-service/tasks/main.yml

```diff
@@ -29,87 +47,141 @@
   include_role:
     name: nginx_vhost
   vars:
-    appname: "image-service"
-    hostname: "{{ images_hostname }}"
-    context_path: "{{ images_context_path }}"
+    hostname: "{{ image_service_hostname }}"
+    context_path: "{{ image_service_context_path }}"
     nginx_paths:
-      - path: "{{ images_context_path }}"
+      - path: "{{ image_service_context_path }}"
         sort_label: "1_ws"
         is_proxy: true
-        proxy_pass: "http://127.0.0.1:8080{{ images_context_path }}"
```

Comment:
Using `8080` conflicts with servers with tomcat installed (like demo playbook). 

I think you should use other different port like CAS services do:
https://github.com/AtlasOfLivingAustralia/ala-install/blob/1bb42eb49caa3a8246686fb1149856e80e81fc05/ansible/aws-cas-5.yml#L181